docs/src/simulation_tutorial.md

```diff
@@ -218,59 +218,107 @@ kb07_m.σ
 298.026 /  680.032 =  `0.438252`
 
 ```@example Main
 kb07_m.θ
 ```
 
-We can not calculate the `θ` for variance component *`item - prec: maintain`* yet, because it includes the correlation of
-*`item - prec: maintain`* and *`item - (Intercept)`*.
+We can not calculate the `θ`s for variance component *`item - prec: maintain`* this way, because it includes the correlation of
+*`item - prec: maintain`* and *`item - (Intercept)`*. But keep in mind that the relation of  *`item - prec: maintain`*-variability (`252.521`)
+and the `residual`-variability (`680.032`) is 252.521  /  680.032 =  `0.3713369`.
+
 The `θ` vector is the flattened version of the variance-covariance matrix - a lowertrinangular matrix.
-The on-diagonal elements are just the standard deviations (the `σ`'s), If all off-diagonal elements are zero, we can use our
+The on-diagonal elements are just the standard deviations (the `σ`'s). If all off-diagonal elements are zero, we can use our
 calculation above. The off-diagonal elements are covariances and correspond to the correlations (the `ρ`'s).
-If they are unequal to zero, as it is in our `kb07`-dataset, we cannot recreate the variance-covariance matrix having the model output.
-We just take it from the model we already fitted
+If they are unequal to zero, as it is in our `kb07`-dataset, one way to get the two missing `θ`-values is to take the values directly from the model we have already fitted.
 
 See the two inner values:
 ```@example Main
 kb07_m.θ
 ```
 
-### Play with theta
-TODO: explain this section
-```@example Main
-# if you don't do a deepcopy, then you just have two names for the same model
-# instead of a (full) copy
-kb07_m2 = deepcopy(kb07_m)
+Another way is to make use of the 'create_re()' function.
+Here you have to define the relation of all random effects variabilities to the variability of the residuals, as shown above,
+and the correlation-matrices.
 
-# the diagonal is always 1 because everything is perfectly correlated with itself
-# the elements below the diagonal follow the same form as the `Corr.` entries
-# in the output of VarCorr
-# the elements above the diagonal are just a mirror image
+Lets start by defining the correlation matrix for the `item`-part.
+
+The diagonal is always `1.0` because everything is perfectly correlated with itself.
+The elements below the diagonal follow the same form as the `Corr.` entries in the output of `VarCorr()`. 
+In our example the correlation of
+*`item - prec: maintain`* and *`item - (Intercept)`* is `-0.7`.
+The elements above the diagonal are just a mirror image.
+
+```@example Main
 re_item_corr = [1.0 -0.7; -0.7 1.0]
+```
 
-re_item = create_re(0.5363168233715857,0.37133693708531357; corrmat=re_item_corr)
-re_item
+Now we put together all relations of standard deviations and the correlation-matrix for the `item`-group:
 
-re_subj = create_re(0.4382528181348316)
+TODO: How is the result called exactly? variance-covariance matrix? 
```

Comment:
I think ["covariance factor" works](https://repsychling.github.io/MixedModelsSim.jl/dev/#MixedModelsSim.create_re-Tuple), but this is where @dmbates keeps us honest.